backend/tests/reservation/getAll.js

```diff
@@ -1,12 +1,65 @@
 import chai, { assert } from 'chai';
 import chaiHttp from 'chai-http';
 import config from '../../config/config';
+import connection from '../../database';
 
 chai.use(chaiHttp);
 
-// Testing /reservation/all GET endpoint
+// before(async () => {
+//   const { error, result } = await connection.asyncQuery(
+//     `INSERT INTO restaurant_db.RESTAURANT VALUES (300, "Example resteraunt");`
+//   );
 
+//   console.log('Inserting starting reservations');
+//   console.log({ error, result });
+// });
+// Testing /reservation/all GET endpoint. Using ids 0-10
 describe('GET reservations/all', () => {
+  it.only('4. should return reservation.', function(done) {
+    // const { error, result } = await connection.asyncQuery(
+    //   `BEGIN;
+    //   INSERT INTO restaurant_db.RESTAURANT VALUES (300, "Example resteraunt");
+    //   INSERT INTO restaurant_db.TABLE VALUES (300, 300, 1, 6);
+    //   INSERT INTO restaurant_db.USER VALUES (300, "First name", "Last name", "09 123,456", "example@email.com");
+    //   INSERT INTO restaurant_db.RESERVATION (ID, Date, Time, Notes, NumberOfGuests, TableID, RestaurantID, UserID)
+    //     VALUES (300, '2020-03-14', '11:30:00', '', '2', '300', '300', '300');
+    //   COMMIT;`
+    // );
```

Comment:
Could you please remove the commented lines of code?

Reply:
Done